### DISCUSSION TITLE
I can generate dis text discussions with chat, with chat say but I cannot generate images, and an error is provided

### DISCUSSION THREAD

**Discussion by therealrobster** [mudler/LocalAI#2307] [General]
I Have Installed the Models, and I am running from the single executable on Linux.

I haven't installed in `~Applications/LocalAI/`

When I try and run the image generation, I get the following error in the terminal:

```
ERR Server error error="grpc process not found: /tmp/localai/backend_data/backend-assets/grpc/diffusers. some backends(stablediffusion, tts) require LocalAI compiled with GO_TAGS" ip=127.0.0.1 latency=67.273045ms method=POST status=500 url=/v1/images/generations
```

The interface gives me this error : `grpc process not found: /tmp/localai/backend_data/backend-assets/grpc/diffusers. some backends(stablediffusion, tts) require LocalAI compiled with GO_TAGS`

I have the following models installed:
 - dreamshaper
 - stablediffusion-cpp

Any help would be greatly appreciated.

Thank you

**Comment by JasonSecurity** [comment]
Same problem on 19/01/2025
